hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/command/ws/server/ConnectedFeeds.java

```diff
@@ -70,27 +74,27 @@ public Session getSession(String feedId) {
     }
 
     @Lock(LockType.READ)
-    public Set<String> getAllFeeds() {
+    public Set<String> getAllSessionIds() {
         return new HashSet<String>(this.sessions.keySet());
     }
 
     /**
-     * Adds the given session and associates it with the given feed ID.
-     * If there is already a session associated with the given feed ID,
-     * the given session is closed, it is not added or associated with the given feed ID
-     * and a error will be logged. The original session remains associated with the feed ID.
+     * Adds the given session and associates it with the given feed ID. If there is already a session associated with
+     * the given feed ID, the given session is closed, it is not added or associated with the given feed ID and a error
+     * will be logged. The original session remains associated with the feed ID.
      *
      * @param feedId the feed ID that will be associated wit the new session
      * @param newSession the new session to add
      * @return if the session was added true is returned; false otherwise.
```

Comment:
I hope that the `addSession()` now fulfills all JavaDoc and functional requirements: https://github.com/hawkular/hawkular-command-gateway/pull/14/files#diff-ab60210134852d097a2667491cc0e008R64 and how it is used: https://github.com/hawkular/hawkular-command-gateway/pull/14/files#diff-a92bf66bf886f02bf5bdf724e8d7cabfR45 and https://github.com/hawkular/hawkular-command-gateway/pull/14/files#diff-9ec1058168d58de9423ea04a103c12a5R51